documentation/bolt_examples.md

```diff
@@ -123,3 +123,221 @@ To convert the `restart_service.ps1` script to a task, giving you the ability to
     ![](bolt_PS_3.png)
 
     **Note:** `--nodes windows` refers to the name of the group of target nodes that you specified in your inventory file. For more information, see [Specify target nodes](bolt_options.md#).
+
+
+## Deploy a package with Bolt and Chocolatey
+
+You can use Bolt with Chocolatey to deploy a package on a Windows node by writing a Puppet plan that installs Chocolatey and uses the Chocolatey package provider to install a package. This is all done using content from the Puppet Forge.
+
+**Before you begin** 
+
+- Install Bolt and Puppet Development Kit (PDK).
+- Ensure you have Powershell and Windows Remote Management (WinRM) access.
+
+In this example, you:
+
+- Build a project-specific configuration using a Bolt project directory and PDK.
+- Download module content from the Puppet Forge.
+- Write a Bolt plan to apply Puppet code and orchestrate the deployment of a package resource using the Chocolatey provider.
+
+### 1. Build a project-specific configuration using a Bolt project directory Bolt runs in the context of a project directory. This directory contains all of the configuration, code, and data loaded by Bolt.
+    
+1. Create a module using the PDK command `pdk new module puppet_choco_tap` 
+
+2. To make the module a Bolt project directory, add a `bolt.yaml` file. 
```

Comment:
I don't think we should promote this workflow. It works for some cases but breaks the "things behave properly as long as your inside your project" since a module will exist inside another project eventually.

What does creating a module with PDK buy us?